src/structures/game_session.ts

```diff
@@ -252,6 +252,7 @@ export default class GameSession {
      */
     endSession = async (): Promise<void> => {
         this.finished = true;
+        deleteGameSession(this.guildID);
```

Comment:
Does deleting the game session this early interfere with any accesses to it in the rest of this function?

Reply:
No, it doesn't disappear until it's garbage collected